firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -124,6 +124,10 @@ static current_control_state current_ctrl_state;
 static PID pid_params;
 
 
+//
+static uint8_t torque_sum;
```

Comment:
No but thats all we have allocated in the struct. I am loathe to change the report message struct but I guess we could.